client/src/api/boxes.js

```diff
@@ -27,3 +27,13 @@ export async function deleteBoxByTitle(title) {
   const answer = await response.text();
   return answer;
 }
+
+export async function addItemByTitle(item, title) {
+  const response = await fetch(`/api/boxes/${title}`, {
+    method: "POST",
+    body: JSON.stringify({ item }),
+    headers: { "Content-Type": "application/json" },
+  });
+  const newItem = await response.text();
```

Comment:
`newItem` is equal with `Successfully create the new item ${newItem.item}`. 
So it looks like a success message.

You don't display this message, so I guess, you don't really need it.

Maybe it makes more sense to return the new box.